README.md

```diff
@@ -161,6 +161,7 @@ Name          | Types  | Description
 `extension`   | String | Name of the extension to install the database                                    | Name of resource | yes
 `version`     | String | Version of the extension to install                                              |                  | no
 `old_version` | String | Older module name for new extension replacement. Appends FROM to extension query |                  | no
+`system_user` | String | Define the user on the system to execute sql command                             | Not set          | no
```

Comment:
```suggestion
`system_user` | String | Define the user on the system to execute sql command                             |           | no
```

Reply:
Currently, the default user for Postgres extension is `Postgres`. I just add the ability to provide a user. If it's not provided then default `Postgres` as before.

Reply:
We leave this field blank to denote this

Reply:
I am afraid that I could not understand what we are trying to denote here by not providing the info.

Reply:
I have made the changes but still I would love to hear reason behind it.